package.json

```diff
@@ -53,7 +53,7 @@
     "bedrock-jsonld-document-loader": "^1.0.1",
     "bedrock-passport": "^6.0.0",
     "bedrock-permission": "^3.2.0",
-    "bedrock-profile": "^7.1.0",
+    "bedrock-profile": "github:digitalbazaar/bedrock-profile#issue-30",
```

Comment:
provided that this PR is still needed, can we swap the branches out for releases of `bedrock-profile`? If we can this should be good to go.

Reply:
Updated here 2dd6e9e